datasets/gov/lsd/pajamu_ir_gyvenimo_salygos.csv

```diff
@@ -0,0 +1,404 @@
+,,,,,,,,,5,,,,Dirbtų valandų skaičius prilygsta visai darbo savaitei  ,
+,,,,,,,,,6,,,,"Esu namų šeimininkas (-ė), prižiūriu vaikus arba kitus asmenis  ",
+,,,,,,,,,7,,,,Dėl kitų priežasčių  ,
+,,,,,pl130,string,,,,4,open,,"Darbuotojų skaičius įstaigoje, įmonėje, šeimos versle",
+,,,,,,enum,,,1,,,,"Tikslus skaičius, jeigu yra nuo 1 iki 10",
+,,,,,,,,,2,,,,"Tikslus skaičius, jeigu yra nuo 1 iki 10",
+,,,,,,,,,3,,,,"Tikslus skaičius, jeigu yra nuo 1 iki 10",
+,,,,,,,,,4,,,,"Tikslus skaičius, jeigu yra nuo 1 iki 10",
+,,,,,,,,,5,,,,"Tikslus skaičius, jeigu yra nuo 1 iki 10",
+,,,,,,,,,6,,,,"Tikslus skaičius, jeigu yra nuo 1 iki 10",
+,,,,,,,,,7,,,,"Tikslus skaičius, jeigu yra nuo 1 iki 10",
+,,,,,,,,,8,,,,"Tikslus skaičius, jeigu yra nuo 1 iki 10",
+,,,,,,,,,9,,,,"Tikslus skaičius, jeigu yra nuo 1 iki 10",
+,,,,,,,,,10,,,,"Tikslus skaičius, jeigu yra nuo 1 iki 10",
+,,,,,,,,,11,,,,Nuo 11 iki 19 asmenų,
+,,,,,,,,,12,,,,Nuo 20 iki 49 asmenų,
+,,,,,,,,,13,,,,50 ar daugiau asmenų,
+,,,,,,,,,14,,,,Mažiau negu 11 asmenų,
+,,,,,,,,,15,,,,Daugiau negu 10 asmenų,
+,,,,,pl140,string,,,,4,open,,Dabartinio ar paskutinio pagrindinio darbo sutarties tipas,
+,,,,,,enum,,,1,,,,Nuolatinis darbas/neterminuota darbo sutartis,
+,,,,,,,,,2,,,,Laikinas darbas/terminuota darbo sutartis,
+,,,,,pl150,string,,,,4,open,,Ar turite pavaldinių dabartiniame pagrindiniame darbe (turėjote paskutiniame pagrindiniame darbe)? ,
+,,,,,,enum,,,1,,,,Taip,
+,,,,,,,,,2,,,,Ne,
+,,,,,py010g,number,,,,4,open,,Metinės samdomojo darbo pajamos,
+,,,,,py010n,number,,,,4,open,,"Metinės samdomojo darbo pajamos, atskaičius mokesčius",
+,,,,,py020g,number,,,,4,open,,Metinės pajamos iš darbdavio natūra,
+,,,,,py021g,number,,,,4,open,,"Metinė piniginė nauda, gauta, naudojantis darbdavio automobiliu asmeniniams tikslams",
+,,,,,py030g,number,,,,4,open,,Metinės darbdavio socialinio draudimo įmokos,
+,,,,,py035g,number,,,,4,open,,Papildomos metinės įmokos į privačius pensijų draudimo fondus,
+,,,,,py050g,number,,,,4,open,,Metinės savarankiško darbo pajamos,
+,,,,,py050n,number,,,,4,open,,"Metinės savarankiško darbo pajamos, atskaičius mokesčius",
+,,,,,py070g,number,,,,4,open,,"Metinė suvartotų produktų, gautų iš savo asmeninio ūkio, vertė",
+,,,,,py080g,number,,,,4,open,,Metinės privačių pensijų fondų ar pensijų draudimo išmokos,
+,,,,,py090g,number,,,,4,open,,Metinės socialinės išmokos bedarbiams,
+,,,,,py090n,number,,,,4,open,,"Metinės socialinės išmokos bedarbiams, atskaičius mokesčius",
+,,,,,py100g,number,,,,4,open,,Metinės socialinės išmokos senatvėje,
+,,,,,py110g,number,,,,4,open,,"Metinės socialinės išmokos našliams, našlaičiams",
+,,,,,py120g,number,,,,4,open,,Metinės socialinės išmokos ligos atveju,
+,,,,,py120n,number,,,,4,open,,"Metinės socialinės išmokos ligos atveju, atskaičius mokesčius",
+,,,,,py130g,number,,,,4,open,,Metinės socialinės išmokos negalios atveju,
+,,,,,py140g,number,,,,4,open,,Metinės socialinės išmokos mokymuisi (stipendijos),

```

Comment:
```suggestion
,,,,,py140g,number,,,,2,open,,Metinės socialinės išmokos mokymuisi (stipendijos),
,,,,,,comment,type,,"update(type: ""money"", ref: ""EUR"")",4,,spinta:40,,
```

Reply:
Šis ir buvę komentarai pataisyti dėl valiutos.